tags.go

```diff
@@ -0,0 +1,13 @@
+package shopify
+
+import "strings"
+
+// SplitTags parses a shopify tag string into a list of the individual tags
+func SplitTags(tags string) []string {
+	return strings.Split(tags, ", ")
+}
+
+// ToTags combines a list of tags into a single shopify tag string
+func ToTags(tags []string) string {
+	return strings.Join(tags, ", ")
+}
```

Comment:
these could becomes methods on a new type, but that would probably be overkill

Reply:
Yeh, so that's the big debate. 

1. Leave as []string
2. Add a type that we can split off (like above)
3. Change to string (as shopify) and then we have to split ourselves outside of the package.

Reply:
I think it makes sense for the functions to be here because it's about the data types that shopify uses.